export.md

```diff
@@ -29,11 +29,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Bulk Data Export Requests
 
+All Bulk Data Export Requests MUST include a valid access token in the ```Authorization``` header (i.e., ```Authorization:  Bearer {{token}}```).  See the Security Considerations section above.  
```

Comment:
See https://github.com/smart-on-fhir/fhir-bulk-data-docs/pull/95/files#r241950203; I think we should remove this provision.